bullet_train-super_scaffolding/config/locales/zh-CN/scaffolding/completely_concrete/tangible_things.zh-CN.yml

```diff
@@ -0,0 +1,224 @@
+zh-CN:
+  scaffolding/completely_concrete/tangible_things: &tangible_things
+    label: &label 有形物品
+    breadcrumbs:
+      label: *label
+    navigation:
+      label: *label
+      icon: fal fa-puzzle-piece
+    buttons: &buttons
+      new: 添加新有形物品
+      create: 创建有形物品
+      edit: 编辑有形物品
+      update: 更新有形物品
+      destroy: 删除有形物品
+      shorthand:
+        edit: 编辑
+        destroy: 删除
+      confirmations:
+        # TODO 根据您的用例自定义。
+        destroy: 确定要删除%{tangible_thing_name}吗？这也将删除所有子资源，此操作无法撤消。
+    fields: &fields
+      id:
+        heading: Tangible Thing ID
+      absolutely_abstract_creative_concept_id:
+        heading: Creative Concept ID
+      # 🚅 skip this section when scaffolding.
```

Comment:
Should we translate this string? I haven't tested this against Super Scaffolding yet, but I saw that other strings such as "# 🚅 停止我们现在正在进行的任何跳转。" were translated.

Reply:
The hooks for transforming are [here](https://github.com/bullet-train-co/bullet_train-core/blob/d1d9be164aa8cfd4a7b06c456f5d0fc2e4529f12/bullet_train-super_scaffolding/lib/scaffolding/transformer.rb#L49-L68) in case you want to check them out.

Reply:
Thanks for your comment, I will restore some of this content